cmd/rm-main.go

```diff
@@ -220,27 +241,35 @@ func removeRecursive(url string, isIncomplete bool, isFake bool, older int) erro
 		}
 
 		urlString := content.URL.Path
-		printMsg(rmMessage{
-			Key:  targetAlias + urlString,
-			Size: content.Size,
-		})
+		// print the rmMessage only if content is a bucket or object. clnt.List lists
+		// prefixes as well which unifies the S3 listing with folder listing - however
+		// these prefixes are not actual objects on s3 and the content.Time defaults to
+		// zero. Filter these out to prevent erroneous rmMessage being printed to console.
```

Comment:
You can skip entries those are ending with `/` and having zero time for completeness.

Reply:
done.